### PR TITLE
Master

### PR DESCRIPTION
**Pull request by kylefoo**

### CODE REVIEW

**Comment by jimmynguyc** [comment]
@kylefoo can fix conflicts?

File: README.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ This handbook will be for Kuala Lumpur, the capital city of Malaysia, for the ti
 * If you hail a cab, ask them to turn on the meter or use GrabTaxi
 
 ## Food
-* Street food for the win @JalanAlor! https://youtu.be/OzkC8HRjHw0
-* https://www.tourplus.my/en/attractions/foot-hunt-in-jalan-alor
+* Street food for the win @JalanAlor! [https://youtu.be/OzkC8HRjHw0]
+* (Jalan Alor)[https://www.tourplus.my/en/attractions/foot-hunt-in-jalan-alor]
```
Comment (jimmynguyc) [inline README.md]: I think you terbalik the markdown syntax :D

I think its `[`Jalan Alor`]``(`https://www.tourplus.my/en/attractions/foot-hunt-in-jalan-alor `)`

Comment (kylefoo) [inline README.md]: @jimmynguyc alright, sry abt that.

**Comment by kylefoo** [comment]
@jimmynguyc pls review latest fixes

**Comment by jimmynguyc** [comment]
👍

**Pull request merged**